### PR TITLE
Create instructions to install UCX v0.9.0 and greater from a Databricks cluster web terminal

### PR DESCRIPTION
**Pull request by dmoore247**
Stumbled upon how to install UCX from a web terminal. This will be useful for when the user does not have desktop access to install dependencies etc.

image

### CODE REVIEW

**Comment by dmoore247** [comment]
I need to format the readme better... working...

File: docs/cluster_install.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Verify Databricks cli:
+
+```shell
+which databricks
+databricks --version
+```
+
+Configure Databricks cli to authenticate to correct workspace, add host name and PAT token:
+```shell
+databricks configure
```
Comment (nfx) [inline docs/cluster_install.md:35]: This is security hole, as this configuration is accessible to all users on the same cluster. No way we're recommending this

**Review by nfx — CHANGES_REQUESTED** [review]

File: docs/cluster_install.md (hunk 1 above)
Comment (nfx) [reply docs/cluster_install.md:35]: cc @hamza-db

Comment (dmoore247) [reply docs/cluster_install.md:35]: We can specify a single user cluster and denying access to the cluster.

Comment (dmoore247) [reply docs/cluster_install.md:35]: @nfx any alternatives you'd suggest?

Comment (dmoore247) [reply docs/cluster_install.md:35]: Ok, Environmental variable is session based, not available to other shells.

Comment (nfx) [reply docs/cluster_install.md:35]: users by mistake run it on a shared cluster and then it's root - everyone, they leak admin credentials for all users. no.

**Comment by nfx** [comment]
this PR has security windows wide open, not willing to be on customer calls related to breached PAT tokens on shared clusters, that were not recommended explicitly. let's wait for a proper implementation on platform side.

**Pull request closed without merging**